build.assets/Makefile

```diff
@@ -328,3 +277,28 @@ release-arm64-fips: buildbox-arm-fips
 	@if [ -z ${VERSION} ]; then echo "VERSION is not set"; exit 1; fi
 	docker run $(DOCKERFLAGS) $(BCCFLAGS) -i $(NOROOT) $(BUILDBOX_ARM_FIPS) \
 		/usr/bin/make -C e release -e ADDFLAGS="$(ADDFLAGS)" OS=$(OS) ARCH=$(ARCH) RUNTIME=$(RUNTIME) FIPS=yes VERSION=$(VERSION) GITTAG=v$(VERSION)
+
+#
+# Run docs tester to detect problems.
+#
+.PHONY: docsbox
+docsbox:
+	docker pull $(DOCSBOX)
+
+.PHONY: test-docs
+test-docs: MAKEFILE_DIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
```

Comment:
I may be wrong, but won't `make` switch CWD to the directory of the makefile?
in that case, can this be `$(shell pwd)`?

Reply:
https://stackoverflow.com/a/18137056/11329621 seems to suggest you need to do a bit more than that to make it reliable

Reply:
This is removed now anyway.